spec/controllers/registered_applications_controller_spec.rb

```diff
@@ -35,6 +35,8 @@
         get :show, params: {id: my_app.id}
         expect(assigns(:registered_application)).to eq(my_app)
       end
+      
```

Comment:
I put that in by analogy from the Bloccit Checkpoint 19 spec for PostsController.show.

```
   describe "GET show" do
     it "returns http success" do
 # #16
       get :show, {id: my_post.id}
       expect(response).to have_http_status(:success)
     end
     it "renders the #show view" do
 # #17
       get :show, {id: my_post.id}
       expect(response).to render_template :show
     end
 
     it "assigns my_post to @post" do
       get :show, {id: my_post.id}
 # #18
       expect(assigns(:post)).to eq(my_post)
     end
   end
```